src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tests
     public class DebugTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
         public void Asserts()
         {
             VerifyLogged(() => { Debug.Assert(true); }, "");
```

Comment:
These tests are not working on desktop, because we have added an internal class WindowsDebugLogger that's not there in desktop implementation.. the tests are using an internal field of this class type to test, and the product lib is allowing InternalsVisibleTo to enable testing..